include/hipSYCL/sycl/libkernel/accessor.hpp

```diff
@@ -2231,6 +2231,23 @@ class local_accessor
 
   local_accessor() = default;
 
+  template <typename U,
+  typename = std::enable_if_t<
+  std::is_const_v<dataT> &&
+  std::is_same_v<U, std::remove_const_t<dataT>>>>
```

Comment:
Can you please indent code that is part of template parenthesis? (see other template functions in this class as examples) :)